plotly/tools.py

```diff
@@ -2871,3 +2891,149 @@ def get_candle_decrease(self):
         return (decrease_x, decrease_close, decrease_dif,
                 stick_decrease_y, stick_decrease_x)
 
+class _Dendrogram(FigureFactory):
+    ''' Returns a Dendrogram figure object
+        Example usage:
+        D = Dendrogram( Z )
+        fig = { 'data':D.data, 'layout':D.layout }
+        py.iplot( fig, filename='Dendro', validate=False )'''
+    
+    def __init__(self, X, orientation='bottom', labels=None, colorscale=None, \
+                 width=700, height=700, xaxis='xaxis', yaxis='yaxis' ):
```

Comment:
thoughts on keeping `width` and `height` out of the defaults and using the `autosize` default instead? I much prefer the full-width view inside ipython notebooks:

![image](https://cloud.githubusercontent.com/assets/1280389/9071441/52d5b89e-3ac3-11e5-8f89-1bdb2d25883c.png)